web_app/app.py

```diff
@@ -32,6 +33,9 @@ class PIDConstants(BaseModel):
 
 app = FastAPI()
 
+# Serve the static react frontend
+app.mount("/static", StaticFiles(directory="public"), name="public")
```

Comment:
Is the directory intended to be `auv_gui/public`?

Running startup.sh yields this error:
`Running app.py Server
Traceback (most recent call last):
  File "/home/kamiyata/yonderdeep/Nautilus/web_app/app.py", line 37, in <module>
    app.mount("/static", StaticFiles(directory="public"), name="public")
  File "/home/kamiyata/.local/lib/python3.10/site-packages/starlette/staticfiles.py", line 56, in __init__
    raise RuntimeError(f"Directory '{directory}' does not exist")
RuntimeError: Directory 'public' does not exist
`

Reply:
Yes it is intended to be that, will fix.